packages/maps/src/lib/themes/bounds.js

```diff
@@ -1,16 +1,53 @@
-export const GREATER_LONDON_ZOOM = 11;
-export const GREATER_LONDON_CENTER = [0.05, 51.5151];
+export const centerOfBounds = (bounds) => {
+	const midpoint = (min, max) => max - (max - min) / 2;
+	return [midpoint(bounds[0][0], bounds[1][0]), midpoint(bounds[0][1], bounds[1][1])];
+};
+
+// scaleBounds returns a new bounds that scales the input bounds up by the
+// specified amount { x, y } with the center as the origin.
+//
+// Given the amounts:
+// - { x: 0.1, y: 0.1 } then both width and height will be 10% bigger;
+// - { x: 0.5, y: 1 } then width will be 50% bigger and height 100% bigger;
+// - { x: 2, y: 0 } then width will be 200% bigger and height left unchanged;
+// - etc.
+export const scaleBounds = (bounds, amount = { x: 0, y: 0 }) => {
+	const width = bounds[0][0] - bounds[1][0];
+	const height = bounds[0][1] - bounds[1][1];
+
+	const widthMul = (amount.x || 0) / 2;
+	const heightMul = (amount.y || 0) / 2;
 
+	return [
+		[
+			// Bottom left point
+			bounds[0][0] - Math.abs(width * widthMul),
+			bounds[0][1] - Math.abs(height * heightMul)
+		],
+		[
+			// Top right point
+			bounds[1][0] + Math.abs(width * widthMul),
+			bounds[1][1] + Math.abs(height * heightMul)
+		]
+	];
+};
+
+export const GREATER_LONDON_ZOOM = 11;
 export const GREATER_LONDON_BOUNDS = [
-	[-0.489, 51.28],
-	[0.236, 51.686]
+	[-0.505, 51.285],
+	[0.33, 51.693]
```

Comment:
```suggestion
	[-0.517, 51.278],
	[0.342, 51.698]
```


Where do these new values come from?

Running one of these SQL queries:

```sql
select st_transform(ST_Envelope(shape), 4326) from gisapdata.core_gla_boundary;
select st_transform(ST_SetSRID(ST_extent(geom), 27700), 4326) from gisapdata.core_london_borough;
```

indicates that the corners of a bounding-box would be:
```
-0.5160678730533503
 0.3413716859178191

51.27878314684388
51.69760660683185
```
Rounding to 3 d.p., by rounding the lower-bounds down and the upper-bounds up (i.e. both away from 0) gives:

```
-0.517
 0.342

51.278
51.698
```

The currently propsoed bound sclip part of Havering and Hillingdon:

```sql
select * from gisapdata.core_london_borough where not st_coveredBy(geom, st_transform(ST_MakeEnvelope(-0.505, 51.285, 0.33, 51.693, 4326), 27700));
-- returns Havering and Hillingdon
```

Whereas the bounds suggested in this comment don't:

```sql
select * from gisapdata.core_london_borough where not st_coveredBy(geom, st_transform(ST_MakeEnvelope(-0.517, 51.278, 0.342, 51.698, 4326), 27700));
-- no results

```

Reply:
My intention was to include all of Greater London based on the borough boundary context layer since that is what users will see when the map loads. The original coords cut off a chunk of Havering.

It was a quick eyeball test in mobile view. **You're welcome to generate a more precise set of coords if you like.**

Reply:
> **You're welcome to generate a more precise set of coords if you like**

I have - they're in my Suggested Change above. I'll click the `Commit Suggestion` button on it.